src/include/neorados/RADOS.hpp

```diff
@@ -1147,4 +1019,15 @@ struct hash<neorados::IOContext> {
 };
 } // namespace std
 
+namespace fmt {
+template <>
+struct formatter<neorados::Object> : ostream_formatter {};
+
+template <>
+struct formatter<neorados::IOContext> : ostream_formatter {};
+
+template <>
+struct formatter<neorados::Op> : ostream_formatter {};
+} // namespace fmt
```

Comment:
this appears to be the cause of [centos9 build failures in shaman](https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos9,DIST=centos9,MACHINE_SIZE=gigantic/68579/consoleFull):
```
src/neorados/RADOS.cc:38:
src/include/neorados/RADOS.hpp:1024:56: error: expected class-name before ‘{’ token
 1024 | struct formatter<neorados::Object> : ostream_formatter {};
      |                                                        ^
src/include/neorados/RADOS.hpp:1027:59: error: expected class-name before ‘{’ token
 1027 | struct formatter<neorados::IOContext> : ostream_formatter {};
      |                                                           ^
src/include/neorados/RADOS.hpp:1030:52: error: expected class-name before ‘{’ token
 1030 | struct formatter<neorados::Op> : ostream_formatter {};
      |                                                    ^
```